src/cli/README_DATASET.md

```diff
@@ -107,6 +107,58 @@ After the device successfully attaches to a Thread network, the device will retr
    Done
    ```
 
+### Using the Dataset Updater to update Operational Dataset
+
+Dataset Updater can be used for a delayed update network parameters on all devices of a Thread Network.
+
+1. Clear the dataset buffer and add the Dataset fields to update.
+
+   ```bash
+   > dataset clear
+   Done
+
+   > dataset channel 12
+   Done
+   ```
+
+2. Set the delay timer parameter (example uses 5 minutes or 300000 ms). Check the resulting dataset. No need to specify active or pending timestamps as Dataset Updater will handle this. If specified the `dataset updater start` will give error.
+
```

Comment:
suggest:  ....There is no need to specify active or pending timestamps because  the Dataset Updater will handle this. If specified the `dataset updater start` will issue an error.